src/test/Operate.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import operate from '../logic/operate';
+import Header from '../components/nav';
+import Calculator from '../pages/Calculator';
+
+// test('Render Home and check quote text', async () => {
+//   render(<App />, { wrapper: BrowserRouter });
+//   const user = userEvent.setup();
+
+//   expect(screen.getByText(/calculator/i)).toBeInTheDocument();
+
+//   await user.click(screen.getByText(/quote/i));
+//   expect(screen.getByText(/- William Thurston/i)).toBeInTheDocument();
+// });
+
+test('Render NavBar', () => {
+  const renderHeader = renderer.create(<BrowserRouter><Header /></BrowserRouter>).toJSON();
+  expect(renderHeader).toMatchSnapshot();
+});
+
+test('Render Calculator', () => {
+  const renderCalculator = renderer.create(<Calculator />).toJSON();
+  expect(renderCalculator).toMatchSnapshot();
+});
```

Comment:
- [ ] These tests should not be in the ```operate.js``` test file. Kindly ensure this test file contains only tests relating to ```operate.js```
- [ ] Also, according to the requirements, ensure that you write tests for all the functionalities & scenarios of the ```operate.js``` file

Reply:
@Ghiftee Thank you for the review! I'm trying as hard as I can, my partner left me yesterday and I was late with this project, I'm very stressed, so now I will send you again the changes that you asked me to make. If you find something else, please help me a little bit more, because this is very hard for me, anyways I think I'm done and I don't want to bother you, thank you again, and nice coding!